src/vsr/superblock.zig

```diff
@@ -1208,6 +1213,7 @@ pub fn SuperBlockType(comptime Storage: type) type {
             } else if (context.copy == stopping_copy_for_sequence(superblock.working.sequence)) {
                 @panic("superblock free set lost");
             } else {
+                log.debug("open: read_free_set: broken copy={}", .{ context.copy });
```

Comment:
```suggestion
                log.debug("open: read_free_set: corrupt copy={}", .{ context.copy });
```

Reply:
We can't distinguish between torn writes through crash, or corruption, or updates-in-place. Figured we go with `corrupt` then to line up with our fault model.